web/.htaccess

```diff
@@ -5,6 +5,10 @@
 # to each configured DirectoryIndex file (e.g. index.php, index.html, index.pl).
 DirectoryIndex app.php
 
+# Disable the MultiViews option preventing from using route paths beginning with
+# "/app", "/app_dev" or "/config".
+Options -MultiViews
```

Comment:
You probably need to test whether the module is enabled at all. `<IfModule mod_negotiation.c>`